libass/ass_font.c

```diff
@@ -253,8 +253,11 @@ size_t ass_font_construct(void *key, void *value, void *priv)
     font->size = 0.;
 
     int error = add_face(render_priv->fontselect, font, 0);
-    if (error == -1)
+    if (error == -1) {
+        free(font->desc.family);
         font->desc.family = NULL;
+    }
+    
```

Comment:
Style-Nit: There should be no trailing whitespaces (so no spaces at all on empty lines).

*(GitHub's diff view doesn't diplay any spaces here, but git am warned about it and checking out your branch directly also confirms the presence of spaces. That's a bit odd.)*

Reply:
Should be fixed.